src/CodeStyle/Core/Analyzers/Formatting/Formatter.cs

```diff
@@ -6,12 +6,13 @@
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Formatting;
 using Microsoft.CodeAnalysis.Formatting.Rules;
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
 using OptionSet = Microsoft.CodeAnalysis.Diagnostics.AnalyzerConfigOptions;
 
-namespace Microsoft.CodeAnalysis.Formatting
+namespace Microsoft.CodeAnalysis.CodeStyle.Formatting
```

Comment:
This change is required to avoid use of extern aliases as the same `Microsoft.CodeAnalysis.Formatting.Formatter` is a public type defined in Workspaces layer. For example, see the change in `FormattingCodeFixHelper.cs` below.

Reply:
Let's stick with the `extern alias`.

Reply:
I am fine with using the extern alias for Formatting related code. However, this approach is not viable for Options related types, because they end up with ambiguous types coming from both the CodeStyle assembly and Workspaces assembly in the VB unit test projects, for which there is no resolution as VB does not support extern aliases. Let us discuss this at the meeting if required.

Reply:
Alright, I attempted to revert this change, but it breaks down in code style fixes project and unit test project. We have shared extension methods that reference `Formatter.Annotation`, and the current extern alias based approach forces the use of extern aliases in all these files for C# projects, and there is no way to resolve these ambiguity errors for VB projects. Given that the Formatter implementation inside CodeStyle layer is only for the formatting analyzer, I think it is reasonable to move it to a separate internal namespace and avoid these conflicts in rest of the code in the shared layer.

Also tagging @CyrusNajmabadi, who is hitting similar ambiguity conflicts when attempting to move code in shared layer.

Reply:
I have changed the namespace to `Microsoft.CodeAnalysis.Internal.Formatting` for consistency with other public Workspace types linked into Code Style layer.